src/js/easymde.js

```diff
@@ -1886,8 +1886,12 @@ EasyMDE.prototype.markdown = function (text) {
 
             /* Check if HLJS loaded */
             if (hljs) {
-                markedOptions.highlight = function (code) {
+                markedOptions.highlight = function (code, language) {
+                    if (language != undefined && hljs.getLanguage(language) != undefined) {
```

Comment:
Can do, I was just looking at the surrounding code in this file that sometimes uses unstrict equality as well. So, I thought it is okay here. I am not sure it only undefined or whether it could be null as well. So I want to capture both cases. Should I do `x !== undefined || x !== null` or just `x`?

Reply:
Maybe `!language && !hljs.getLanguage(language)`  works as well, that should catch both `null` and `undefined`